app/job/shared/orderInfo.service.ts

```diff
@@ -12,7 +12,7 @@ export class OrderInfoService {
         switch (statusNumber) {
             case 1:
                 _orderInfoHeading = "Your Order is created, we will start processing it shortly!";
-                _orderInfoDesc = "We've got your order, we are getting it ready, a confirmation of your order will be sent to <b>"+ job.User.Email +".</b>";
+                _orderInfoDesc = "We've got your order, we are getting it ready";
```

Comment:
hard coded text? If there are a lots of text information to show to the user in many different places, may be a separate JSON file is suitable? Just a suggestion, I don't know the Angular way.